User-Help/GlobalFAQ.md

```diff
@@ -0,0 +1,34 @@
+<!--- Codidact FAQ -->
+
+# Codidact FAQ
+
+## What is Codidact?
+
+Codidact is an open-source initiative to provide a high-quality, community-controlled, open source Q&A platform. Visit <https://codidact.org> to learn more about the platform we're building to empower anyone to host a Q&A community, or visit <https://codidact.com> to browse our family of hosted sites. 
+
+## Who runs Codidact?
+
+Users just like you! The software is built and hosted by volunteers who are passionate about the need for a community-driven Q&A site. Each site is moderated by a team of dedicated, experienced users who spend their time weeding out spam, coaching newbies, and ensuring questions are on-topic and high-quality. 
```

Comment:
I'd prefer "newcomers" to "newbies" -- less slang (especially for an international audience), and I've been told some find "newbies" a little off-putting.